docs/install-machine.md

```diff
@@ -40,6 +45,12 @@ To install on Linux, do the following:
 
 			$ docker-machine -v
 			machine version 0.4.0
+			
+## Install from binary
+
+You can install from the  binaries for the various platforms and architectures
+are available at the [Github
+Release](https://github.com/docker/machine/releases/) page.
```

Comment:
Maybe this sentence can be rephrased? 
`You can install from the  binaries for the various platforms and architectures are available at the`